gector/tokenizer_indexer.py

```diff
@@ -21,7 +21,7 @@
 # TODO(joelgrus): Figure out how to generate token_type_ids out of this token indexer.
 
 
-class TokenizerIndexer(TokenIndexer[int]):
+class TokenizerIndexer(TokenIndexer):
     """
     A token indexer that does the wordpiece-tokenization (e.g. for BERT embeddings).
```

Comment:
This docstring is taken from allennlp's [wordpiece_indexer](https://github.com/allenai/allennlp/blob/v0.8.4/allennlp/data/token_indexers/wordpiece_indexer.py#L23). I think we should change this to describe what this indexer is  doing. What I gather that's unique is the whole "offset_mapping" output but I could be wrong.